src/cascadia/TerminalApp/Pane.h

```diff
@@ -120,8 +120,6 @@ class Pane : public std::enable_shared_from_this<Pane>
 
     Borders _borders{ Borders::None };
 
-    std::atomic<bool> _isClosing{ false };

```

Comment:
Is there another way that we're tracking a pane entering the closing state? Is it no longer necessary?

Reply:
The reason I added it initially doesn't exist anymore, hence I removed it. We can also leave it, as it doesn't harm the correctness (closing is irreversible). I am fine with both :blush: